pkg/random/random_test.go

```diff
@@ -20,3 +22,10 @@ func TestConcurrency(t *testing.T) {
 	}
 	wg.Wait()
 }
+
+func TestReader_Read(t *testing.T) {
+	var s []byte = make([]byte, 5)
+	NoBytesRead, err := Reader.Read(s)
+	require.Nil(t, err)
+	require.Equal(t, NoBytesRead, 5)
+}
```

Comment:
Normally gofmt leaves an empty line at the end. Make sure you have properly done the linting